scripts/db_migrator.py

```diff
@@ -899,6 +911,8 @@ def common_migration_ops(self):
         else:
             log.log_notice("Asic Type: {}, Hwsku: {}".format(self.asic_type, self.hwsku))
 
+        self.migrate_route_table_weights()
```

Comment:
since we have other functions as "`migrate_copp_table`", can we have a generic function like `migrate_route_table` so that a future changes can also be contained within this. What do you think?

Reply:
Sure, that's a good idea. Updated now.